components/centraldashboard/public/components/dashboard-view.css

```diff
@@ -31,13 +22,32 @@
         font-family: "Google Sans";
         height: 62px;
         border-bottom: 1px solid var(--divider-color);
-        margin-bottom: .5em;
         @apply --layout-horizontal;
         @apply --layout-center;
     }
+
+    --paper-item-body-secondary: {
+        font-size: 12px;
+        letter-spacing: 0.3px;
+        line-height: 16px;
+        white-space: normal;
+        width: 100%;
+        word-break: break-word;
+    }
 }
 
-.column {grid-row: 1}
+#grid {
+    display: grid;
+    grid-gap: .75em; /* 12px */
+    grid-template-columns: repeat(3, 1fr);
+    margin: 0 auto;
+}
+
+.column {
+    grid-row: 1;
+    min-width: 320px;
+    max-width: 400px;
```

Comment:
If we have this logic defined here, all cards should no longer have this logic, and instead should be:
```css
width: 100%
```

Reply:
Agreed. I think I took care of this already by removing the widths from the `--dashboard-card` mix-in.

Reply:
Ah, I may have missed that. Feel free to resolve